### PR TITLE
docs: update the whole repository

### PR DESCRIPTION
**Pull request by 1170300628**
Add a new classical algorithm.

### CODE REVIEW

**Comment by 1170300628** [comment]
Add the Kruscal MST algorithm.

File: DataStructures/Graphs/KruscalMST.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	/**
+	 * Find the last node in the set.
+	 * 
+	 * @param parent the array of all the parents
+	 * 
+	 * @param index the index you wanted to get
+	 * 
+	 * @return the final result you want to get
+	 */
+	private int find(int parent[], int index) {
```
Comment (yanglbme) [inline DataStructures/Graphs/KruscalMST.java]:
```suggestion
	private int find(int[] parent, int index) {
```

File: DataStructures/Graphs/KruscalMST.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	    while (parent[index] > 0) {
+	        index = parent[index];
+	    }
+	    return index;
+	}
+
+    public static void main(String[] args) {
+        List<Edge> edges = new ArrayList<Edge>();
```
Comment (yanglbme) [inline DataStructures/Graphs/KruscalMST.java]:
```suggestion
        List<Edge> edges = new ArrayList<>();
```

Comment (L170300101) [reply DataStructures/Graphs/KruscalMST.java]: thx  a lot :)

**Pull request closed without merging**